### ISSUE TITLE
python module: get item from id ?

### ISSUE THREAD

**Issue by insatomcat** [markhoerth/dremio_client#60]
Can the python module retrieve a catalog item from its id ?
That's very easy to do with the client, but from the python module I cannot find a way.

For now the only way I found to programmatically get a specific VDS given its ID and path is to start at the root et move throught the path, folder by folder. And this has a limitation given most of my folders have spaces in them, which must be converted to underscores:

```
"path": ["BDF", "VDS Management"]

In [45]: catalog["BDF"]["VDS Management"]
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-45-b5d3437247dc> in <module>
----> 1 catalog["BDF"]["VDS Management"]

KeyError: 'VDS Management'

In [46]: catalog["BDF"]["VDS_Management"]
Out[46]: {"entityType": "folder", "id": "652074a1-7c74-4f25-bc82-e64c32d0cdbb", "path": ["BDF", "VDS Management"], "tag": "vjbOHwWjwZo=", "accessControlList": {"users": null, "groups": null, "version": "1"}}
```

Thank you for your advice.

**Comment by rymurr** [comment]
Hey @insatomcat 

The 'Simple' client has https://github.com/rymurr/dremio_client/blob/master/dremio_client/dremio_simple_client.py#L89 which will fetch by catalog id. Nothing in the object based client though. You might be able to leverage that to get the path and apply that to the catalog. Or there may be an easy way to add the that method to the catalog

**Comment by insatomcat** [comment]
Thanks, I solved my problem using the simple client which was perfect for this usecase.
However I looked into adding the method to the catalog, see #62 which creates an object from an arbitrary cid or path.

**Comment by rymurr** [comment]
fixed in `0.12.1`
